### ISSUE TITLE
Possiblity to make individual cards visible/enabled based on entity

### ISSUE THREAD

**Issue by Bort89** [j-a-n/lovelace-wallpanel#172]
Hi,

Would it be possible to add the possibility to disable/enable individual cards based on a helper (for example) so you can show some cards whenever you want?
For example: when a alarm goes off: only then show a particular card or when it's Halloween, show another card.

Thanks again for this awesome add-on!

**Comment by GeraldG123** [comment]
Hi,

also new to this topic but why not use different profiles here?
And use automation to define which profile is needed?

best regards
Gerald

**Comment by Bort89** [comment]
Maybe I misunderstood the profiles. I thought profiles worked only on the css of the whole panel, not individual cards but maybe I am wrong?

**Comment by kwithus** [comment]
Yes, profiles can be used for this. That’s what I am doing. However, it makes the config very complex and long. I am already past 1000 lines of configuration.
Wallpanel is really very mighty. You can do a lot of different things with this tool. But it’s not very userfriendly to config when you try to use it to max.
In my opinion, the unique selling point of Wallpaper is the screensaver function, which shows different images including the meta-data of the image. All other features, like cards, different profiles etc. are basically something that could be done with any dashboard. 
that’s the reason why I opened another feature request. I would love to have an ‚ordinary‘ dashboard custom-card, which brings the functionality of the screensaver - image -box with meta data, which could be used inside any dashboard.

**Comment by tombul** [comment]
I think this issue can be closed, as the functionality already exists with default home assistant dashboards with visibility and conditions.
Some ways are discussed in the [home assistant community](https://community.home-assistant.io/t/how-to-change-the-visibility-of-a-card-via-an-automation/925783/4) and even mentioned in the [home assistant.io dashboard documentation](https://www.home-assistant.io/dashboards/sections/#adding-sections-and-cards-to-a-sections-view).
That way the cards and visibility conditions can even be configured via visual editor and then pasted into the wallpanel config.
